src/Controller/StudentController.php

```diff
@@ -14,8 +14,8 @@ class StudentController extends AbstractController
     public function index(Request $request)
     {
         $data = [
-            'name' => $request->get('name'),
-            'project' => $request->get('project'),
+            'name' => $request->get('name', 'undefined'),
```

Comment:
Geriau  naudoti kažką lietuvišo (ar pagal projekto kalbą).
Jei atvaizduojama iš Backend `undefined`, tai nėra aišku, ar klaida JavaScript (frontend) pusėje, ar PHP (backend) pusėje.

Plačiau:
 * https://uxplanet.org/how-to-write-a-perfect-error-message-da1ca65a8f36